### PR TITLE
Fix refresh token

### PR DESCRIPTION
**Pull request by sjl1826**
fixed refresh call

### CODE REVIEW

File: Bruin Bite/Network/Moya.swift [hunk 1]
```diff
@@ -106,12 +106,17 @@ extension MainAPI: TargetType {
             return .requestParameters(parameters: ["email": email], encoding: JSONEncoding.default)
         case .matchUser(let user, let meal_times, let meal_day, let meal_period, let dining_halls):
             return .requestParameters(parameters: ["user": user, "meal_times": meal_times, "meal_day": meal_day, "meal_period": meal_period, "dining_halls": dining_halls], encoding: JSONEncoding.default)
+<<<<<<< HEAD
```
Comment (HirdayGupta) [inline Bruin Bite/Network/Moya.swift]: @sjl1826 you’ve left merge conflicts in here - please fix.

File: Bruin Bite/Network/UserManager.swift [hunk 2]
```diff
@@ -144,7 +144,7 @@ class UserManager {
     }
 
     func getNewAccessToken(refreshToken: String) {
-        self.provider.request(.refreshToken(refresh_token: refreshToken)) { result in
+        self.provider.request(.refreshToken(grant_type: "refresh_token", client_id: CLIENTID, client_secret: CLIENTSECRET, refresh_token: refreshToken)) { result in
```
Comment (HirdayGupta) [inline Bruin Bite/Network/UserManager.swift:147]: @sjl1826 can you point me to the docs that say why this will work?

**Review by HirdayGupta — CHANGES_REQUESTED** [review]
Thanks! Some changes requested...

File: Bruin Bite/Network/Moya.swift (hunk 1 above)
Comment (sjl1826) [reply Bruin Bite/Network/Moya.swift]: woops my bad!

File: Bruin Bite/Network/UserManager.swift (hunk 2 above)
Comment (sjl1826) [reply Bruin Bite/Network/UserManager.swift:147]: What will work? Like why do we need to provide grant_type for refresh_token? https://auth0.com/docs/tokens/refresh-token/current

**Review by HirdayGupta — APPROVED** [review]
Looks good! 👍

**Pull request merged**